packages/catala/catala.0.2.0/opam

```diff
@@ -16,7 +16,7 @@ depends: [
   "menhir" {>= "20200211" & < "20211215"}
   "menhirLib" {>= "20200211"}
   "unionFind" {>= "20200320"}
-  "bindlib" {>= "5.0.1"}
+  "bindlib" {= "5.0.1"}
```

Comment:
Although I agree in principle, I'm a bit reluctant to relax constraints here, because we've not been very good at version numbers with Bindlib. For instance, going from `5.0.0` to `5.0.1` introduces breaking changes, it is not just a bug fix release. Also, I'm positive there won't ever be a release between `5.0.1` and `6.0.0` (we're only actively maintaining the latest released version).